presto-main/src/main/java/com/facebook/presto/sql/gen/TryCodeGenerator.java

```diff
@@ -107,6 +110,7 @@ public static MethodDefinition defineTryMethod(
 
         method.comment("Try projection: %s", innerRowExpression.toString());
         method.getBody()
+                .append(wasNull.set(constantBoolean(false)))
```

Comment:
No. This is not a fix for another issue.

This is just part of changing
```
RetType f(..., boolean wasNull) {
  ...
}
```
to
```
RetType f(...) {
  boolean wasNull = false;
  ...
}
```

In the former case, invocation always pass `false` for `wasNull`.

Writing it without ` = false` does not work because byte code generation generally assumes that `wasNull` is initially false.
```
RetType f(...) {
  boolean wasNull;
  ...
}
```